### PR TITLE
Implement roving tabindex on the Image block

### PR DESCRIPTION
**Pull request by diegohaz**
This PR is part of #18619, whose main goal is to implement [roving tabindex](https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex) on the `@wordpress/components`' `Toolbar` component and to use it on the header and block toolbars so they become a single tab stop as recommended by the [WAI-ARIA Toolbar Pattern](https://www.w3.org/TR/wai-aria-practices/#toolbar). Related issues are #15331 and #3383.

This PR continues the work done in #20008, but specifically for the **Image**, **Cover** and **Media & Text** blocks.

GIF showing the Image block while focus is moved using arrow keys and Tab

## How to test

Check Image, Cover and Media & Text blocks toolbars. Use <kbd>Tab</kbd> and arrow keys to move focus.

### CODE REVIEW

File: packages/block-library/src/rich-image/rich-image/index.js [hunk 1]
```diff
@@ -210,57 +209,73 @@ class RichImage extends Component {
...
+														'rotate',
+														{
+															angle: ROTATE_STEP,
+														}
+													),
+											},
+										] }
+									/>
+								) }
+							</ToolbarItem>
```
Comment (diegohaz) [inline packages/block-library/src/rich-image/rich-image/index.js:248]: The diff got messed up. But this is basically wrapping `<DropdownMenu>` with `<ToolbarItem>` and passing the toolbar item props to `toggleProps`:
```jsx
<ToolbarItem>
	{ ( toggleProps ) => (
		<DropdownMenu
			...
			toggleProps={ toggleProps }
		/>
	) }
</ToolbarItem>
```

**Review by gziolo — APPROVED** [review]
There is this features that hide whitespace changes:
Screen Shot 2020-06-04 at 07 18 02

It isn't perfect but it helps to review PRs that contain indentation changes like in here.

It seems like the sooner we add deprecation message the better. #21024 introduced some code that uses the old approach.

**Review by ajlende — APPROVED** [review]
Tested locally, and it works for me!

**Review by ItsJonQ — APPROVED** [review]
🚀 from me! Tested locally, and navigating through blocks via <kbd>Tab</kbd> and Arrows was as expected and advertised :)

Thank you @diegohaz for your continued efforts in improving the Toolbar and a11y! 🙌

**Pull request merged**